cooperator/models/company.py

```diff
@@ -17,6 +17,16 @@ def _compute_base_logo(self):
     subscription_maximum_amount = fields.Float(
         string="Maximum authorised subscription amount"
     )
+    cooperator_number_sequence_id = fields.Many2one(
```

Comment:
We don't need to select the `ir.sequence` for every company, we can use this method, so it applies the multi-company logic automatically, we just need to make sure that the `ir.sequence` with the same `code` is created for all companies.
https://github.com/odoo/odoo/blob/36459d26f1adb92f92d52ce05329e8ad3e95dd91/odoo/addons/base/models/ir_sequence.py#LL274C1-L274C63

Reply:
Very good observation; refactored the logic. Thank you.